--- asterisk-1.8.23.0/Makefile.rules.vanilla	2013-07-16 17:32:56.607940463 +0000
+++ asterisk-1.8.23.0/Makefile.rules	2013-07-16 17:33:09.516206128 +0000
@@ -157,6 +157,6 @@
 
 %: %.o
 	$(ECHO_PREFIX) echo "   [LD] $^ -> $@"
-	$(CMD_PREFIX) $(CXX) $(STATIC_BUILD) -o $@ $(PTHREAD_CFLAGS) $(_ASTLDFLAGS) $^ $(CXX_LIBS) $(ASTLDFLAGS)
+	$(CMD_PREFIX) $(CC) $(STATIC_BUILD) -o $@ $(PTHREAD_CFLAGS) $(_ASTLDFLAGS) $^ $(CC_LIBS) $(ASTLDFLAGS)
 
 dist-clean:: clean
