--- asterisk-1.8.23.0/doc/appdocsxml.dtd.vanilla	2013-07-16 17:32:02.896203786 +0000
+++ asterisk-1.8.23.0/doc/appdocsxml.dtd	2013-07-16 17:32:15.884861880 +0000
@@ -32,7 +32,7 @@
   <!ELEMENT see-also (ref|xi:include)*>
 
   <!ELEMENT ref (#PCDATA)>
-  <!ATTLIST ref type (application|function|astcli|link|manpage|filename|agi) #REQUIRED>
+  <!ATTLIST ref type (application|function|manager|astcli|link|manpage|filename|agi) #REQUIRED>
 
   <!ELEMENT synopsis (#PCDATA)>
 

