# disable dropbear reverse dns lookup

--- dropbear-0.52/options.h.vanilla	2013-01-16 19:14:57.208299893 +0000
+++ dropbear-0.52/options.h	2013-01-16 17:14:05.000000000 +0000
@@ -129,7 +129,7 @@
 /* #define DSS_PROTOK */
 
 /* Whether to do reverse DNS lookups. */
-#define DO_HOST_LOOKUP
+//#define DO_HOST_LOOKUP
 
 /* Whether to print the message of the day (MOTD). This doesn't add much code
  * size */
