--- busybox-1.20.2/init/init.c.vanilla	2012-12-31 13:27:02.827927302 +0000
+++ busybox-1.20.2/init/init.c	2012-12-31 13:36:09.860594684 +0000
@@ -10,7 +10,7 @@
  */
 
 //applet:IF_INIT(APPLET(init, BB_DIR_SBIN, BB_SUID_DROP))
-//applet:IF_FEATURE_INITRD(APPLET_ODDNAME(linuxrc, init, BB_DIR_ROOT, BB_SUID_DROP, linuxrc))
+//applet:IF_FEATURE_INITRD(APPLET_ODDNAME(linuxrc, init, BB_DIR_SBIN, BB_SUID_DROP, linuxrc))
 
 //kbuild:lib-$(CONFIG_INIT) += init.o
 
