--- busybox-1.20.2/init/init.c.vanilla	2013-03-02 16:45:12.291928523 +0000
+++ busybox-1.20.2/init/init.c	2013-03-02 16:49:32.536291191 +0000
@@ -10,7 +10,7 @@
  */
 
 //applet:IF_INIT(APPLET(init, BB_DIR_SBIN, BB_SUID_DROP))
-//applet:IF_FEATURE_INITRD(APPLET_ODDNAME(linuxrc, init, BB_DIR_ROOT, BB_SUID_DROP, linuxrc))
+//applet:IF_FEATURE_INITRD(APPLET_ODDNAME(linuxrc, init, BB_DIR_SBIN, BB_SUID_DROP, linuxrc))
 
 //kbuild:lib-$(CONFIG_INIT) += init.o
 
